README.md

```diff
@@ -50,10 +50,14 @@ Example certificates are included in the repository for test purposes.
 
 ```sh
 $ cd quinn
-$ cargo run --example server -- --cert ../certs/server.chain --key ../certs/server.rsa .
-$ cargo run --example client -- --ca ../certs/ca.der https://localhost:4433/Cargo.toml
+$ cargo run --example server -- --cert ./certs/server.chain --key ./certs/server.rsa ./
+$ cargo run --example client -- --ca ./certs/ca.der https://localhost:4433/Cargo.toml
```

Comment:
I didn't realize cargo would search workspace members for examples! Please also remove the `cd quinn` line so this is consistent.